src/ui/components/map/providers/mapprovider.js

```diff
@@ -90,6 +90,24 @@ export default class MapProvider {
      * @type {boolean}
      */
     this._collapsePins = config.collapsePins || false;
+
+    /**
+     * Locale of the map. Defaults to 'en'
+     * @type {string}
+     */
+    const locale = config.locale || 'en';
```

Comment:
Would it make sense for this to default/fallback to the bundle's locale?

Reply:
Oof, I just saw Tom already made this comment here https://github.com/yext/answers/pull/1070#discussion_r488205215. Totally agree about doing it when the i18n changes are in, in the meantime, would it make sense to use the same default as here in the answers-umd.js? https://github.com/yext/answers/blob/master/src/answers-umd.js#L53

Reply:
Ya, that makes sense! I wasn't aware of that constant

Reply:
After a discussion in slack, we'll default to the locale used in ANSWERS.init() for now